testfiles/CMakeLists.txt

```diff
@@ -621,6 +621,7 @@ ADD_SIMULATION_TEST(IDF_FILE WCE_Diffuse_Shade.idf EPW_FILE USA_IL_Chicago-OHare
 ADD_SIMULATION_TEST(IDF_FILE WCE_DoubleClear_BSDF.idf EPW_FILE USA_IL_Chicago-OHare.Intl.AP.725300_TMY3.epw)
 ADD_SIMULATION_TEST(IDF_FILE WCE_Interior_VB_-45_deg.idf EPW_FILE USA_IL_Chicago-OHare.Intl.AP.725300_TMY3.epw)
 ADD_SIMULATION_TEST(IDF_FILE _5ZoneAirCooled_annual.idf EPW_FILE USA_IL_Chicago-OHare.Intl.AP.725300_TMY3.epw ANNUAL_SIMULATION)
+ADD_SIMULATION_TEST(IDF_FILE _5ZoneAirCooled_LeapYear_annual.idf EPW_FILE MadeUpLeapYear.epw ANNUAL_SIMULATION)
```

Comment:
OK, I just converted the file to the right line endings and pulled in the latest upstream develop and it's all good here again.